cli/README.md

```diff
@@ -26,4 +26,18 @@ CLI supports 2 execution modes:
 * Batch. In this mode all commands will be read from text file or pipe and executed in series. To start this mode run `indy-cli <path-to-text-file>`. Batch mode supports the same commands as interactive mode. Note that by default if some command finishes with the error batch execution will be interrupted. To prevent this start command with `-`. For example, `-wallet create test`. In this case the result of this command will be ignored.
 
 ### Getting help
-The most simple way is just start cli by `indy-cli` command and put `help` command. Also you can look to [Indy CLI Design](../doc/cli-design.md) doc that contains the list of commands and architecture overview.
\ No newline at end of file
+The most simple way is just start cli by `indy-cli` command and put `help` command. Also you can look to [Indy CLI Design](../doc/cli-design.md) doc that contains the list of commands and architecture overview.
+
+### Migration
```

Comment:
I suggest to add a bit more background information:

### Old python-based CLI migration

It is possible to import did's stored in the wallet of deprecated python-based CLI tool. To achieve this user needs to perform the following steps...

Also we need mention that user needs to update old CLI. After this execute script on machine with installed old CLI. After this install new CLI and etc....